backend/video/views.py

```diff
@@ -182,16 +182,29 @@ def list_recent(request):
     # Get the query param from the request, default count is 10
     count = int(request.query_params.get("count", 10))
 
-    # Get a list of videos from recently transcribed videos
+    # Note: Currently, we have implemented this get recent method based on the logic that
+    # one Transcript of either type ORIGINAL_SOURCE or type MACHINE_GENERATED
+    # will always have one video associated with it.
+    # In the future, if that constraint is removed then we might need to alter the logic.
+
+    # Get the 2n latest transcripts from the DB for the user
     recent_transcripts = [
         (transcript.video, transcript.updated_at)
-        for transcript in Transcript.objects.filter(user=request.user.id)
+        for transcript in Transcript.objects.filter(user=request.user.id).order_by(
+            "-updated_at"
+        )[:count]
     ]
 
-    # Get a list of videos from recently translated videos
+    # Get the date of the first trancript from the above list
+    first_transcript_date = recent_transcripts[-1][1]
+
+    # Get the 2n latest translations from the DB for the user
```

Comment:
We are not doing this top-`2n` thing for translations now, right?

Reply:
Will remove.